package.json

```diff
@@ -104,6 +104,7 @@
     "webpack": "4.29.6"
   },
   "dependencies": {
+    "gatsby-plugin-google-tagmanager": "^2.3.11",
```

Comment:
Since this repo is a lerna repo, which means it is a multi-package repo, not a conventional repo, the `gatsby-plugin-google-tagmanager` dependency should only be installed in the `documentation` package. Since none of other packages in this repo use Gatsby, I think we should be installing this dependency in [`packages/documentation/package.json`](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/master/packages/documentation/package.json) instead of the root `package.json`.